.circleci/config.yml

```diff
@@ -49,16 +49,18 @@ jobs:
           name: Pull Deps
           command: |
             source ${INIT_ENV}
-            ./src/CARMANovatelGpsDriver/docker/checkout.sh ${PWD}
+            ./src/CARMANovatelGpsDriver/docker/checkout.bash -r ${PWD}
       - run:
           name: Build Driver
           command: |
             source ${INIT_ENV}
+            export ROS_PARALLEL_JOBS='-j1 -l1' # Try to reduce memory consumption on build
             build-wrapper-linux-x86-64 --out-dir /opt/carma/bw-output bash make_with_coverage.bash -m -e /opt/carma/ -o ./coverage_reports/gcov
       - run:
           name: Run C++ Tests
           command: |
             source ${INIT_ENV}
+            export ROS_PARALLEL_JOBS='-j1 -l1' # Try to reduce memory consumption on build
```

Comment:
Under line 81, i see " cd src/CARMANovatelGpsDriver"

Reply:
yeah that's still the path